tests/utils/local_network_utils.go

```diff
@@ -126,6 +126,12 @@ func RelayMessage(
 		alterMessage,
 	)
 
+	if alterMessage {
```

Comment:
We should be able to instead pass `expectSuccess || alterMessage` to the call to `SendTransactionAndWaitForAcceptance` below